h3-quinn/src/lib.rs

```diff
@@ -203,38 +202,24 @@ impl<S: Session> quic::RecvStream for RecvStream<S> {
         &mut self,
         cx: &mut task::Context<'_>,
     ) -> Poll<Result<Option<Self::Buf>, Self::Error>> {
-        let ret = match self.stream.read_unordered().poll_unpin(cx) {
-            Poll::Pending => Poll::Pending,
-            Poll::Ready(Ok(None)) => Poll::Ready(Ok(None)),
-            Poll::Ready(Err(e)) => return Poll::Ready(Err(e.into())),
-            // If we get the chunk we're looking for, return it right away
-            Poll::Ready(Ok(Some((mut chunk, offset)))) if offset <= self.offset => {
-                chunk.advance((self.offset - offset) as usize); // XXX overflow
-                self.offset += chunk.len() as u64;
-                return Poll::Ready(Ok(Some(chunk)));
+        self.buf.resize(READ_BUF_SIZE, 0);
+
+        match self.stream.read(&mut self.buf).poll_unpin(cx) {
+            Poll::Ready(Ok(Some(n))) => {
+                eprintln!("stream read: {}", n);
```

Comment:
`trace!()` should be used instead.

Reply:
Yea woops, the first thing I did to debug was add these, before eventually adding a `tracing-subscriber` output to the unit tests.